libc/include/math.h.def

```diff
@@ -14,7 +14,22 @@
 #include "llvm-libc-macros/math-macros.h"
 #include "llvm-libc-types/float128.h"
 
+#if !defined(__LIBC_CONST_ATTR) && defined(__cplusplus) && defined(__GNUC__)
```

Comment:
This code is generated by the current headergen so it can't go in the `.h.def` file. Given that the function level attribute system is currently only used by one function it might be reasonable to skip it for the moment and add it later. (CC @SchrodingerZhu)